osm2gtfs/creators/schedule_creator.py

```diff
@@ -0,0 +1,57 @@
+# coding=utf-8
+
+import sys
+import json
+
+
+class ScheduleCreator(object):
+    """
+    The ScheduleCreator loads, validates and prepares configuration data from
+    the schedule source file for further use in the script.
+
+    More information about the standard schedule format:
+    https://github.com/grote/osm2gtfs/wiki/Schedule
+
+    """
+    def __init__(self, config):
+        self.config = config
+
+    def __repr__(self):
+        rep = ""
+        if self.config is not None:
+            rep += str(self.config) + " | "
+        return rep
+
+    def add_schedule_to_data(self, schedule, data):
```

Comment:
The schedule argument will be overridden in the next line and is never passed (see `osm2gtfs.py`).

Reply:
oh, yes, you are right. fixed.